tools/bumper/bumper.go

```diff
@@ -80,18 +87,23 @@ func main() {
 
 			// Create PR
 			exitWithError(fmt.Errorf("create PR not implemented yet"))
+			// update components yaml in the bumping repo instance
+			componentsConfig, err = parseComponentsYaml(configPath)
+			if err != nil {
+				exitWithError(errors.Wrap(err, "Failed to parse components yaml during bump"))
+			}
```

Comment:
componentsConfig is not refreshed when resetting the cnao repo, it's on RAM.
we need to refresh the componentsConfig, otherwise `components.yaml` will accumulate past bump changes.
For example, if first bump was knmstate to 0.32.0, then `componentsConfig` will have this change..
and then next component we need to bump multus to 3.6, the `componentsConfig` will hold both changes..